tests-invoke

```diff
@@ -252,8 +256,21 @@ class PullTask(object):
             cmd = ["timeout", "120m", entry_point]
             if opts.verbose:
                 cmd.append("--verbose")
-            ret = subprocess.call(cmd)
-
+            # Popen instead of call; while not etc... (SubProcess.poll)
+            if ret:
+                sys.stderr.write('Collision detected: {0}\n'.format(ret))
+                return None
+            p = subprocess.Popen(cmd)
+            while p.poll() is None:
+                ret = self.detect_collisions(opts, True)
+                if ret:
+                    sys.stderr.write('Collision detected: {0}\n'.format(ret))
```

Comment:
Let's please make this a different message. "PR got updated, cancelling this test run."?

Reply:
Oh I chose this because it's  similar to the comment on line 225, but I'll add the fact that it's cancelling a running test?